cmd/gaia/cmd/gaiad/main.go

```diff
@@ -44,12 +47,12 @@ func main() {
 		PersistentPreRunE: server.PersistentPreRunEFn(ctx),
 	}
 
-	rootCmd.AddCommand(gaiaInit.InitCmd(ctx, cdc))
-	rootCmd.AddCommand(gaiaInit.CollectGenTxsCmd(ctx, cdc))
-	rootCmd.AddCommand(gaiaInit.TestnetFilesCmd(ctx, cdc))
-	rootCmd.AddCommand(gaiaInit.GenTxCmd(ctx, cdc))
-	rootCmd.AddCommand(gaiaInit.AddGenesisAccountCmd(ctx, cdc))
-	rootCmd.AddCommand(gaiaInit.ValidateGenesisCmd(ctx, cdc))
+	rootCmd.AddCommand(genutilcli.InitCmd(ctx, cdc, app.ModuleBasics))
+	rootCmd.AddCommand(genutilcli.CollectGenTxsCmd(ctx, cdc, app.ModuleBasics, genaccounts.AppModuleBasic{}))
+	rootCmd.AddCommand(genutilcli.GenTxCmd(ctx, cdc, app.ModuleBasics, genaccounts.AppModuleBasic{}, app.DefaultNodeHome, app.DefaultCLIHome))
+	rootCmd.AddCommand(genutilcli.ValidateGenesisCmd(ctx, cdc, app.ModuleBasics))
+	rootCmd.AddCommand(genaccscli.AddGenesisAccountCmd(ctx, cdc))
+	rootCmd.AddCommand(testnet.InitFilesCmd(ctx, cdc, app.ModuleBasics, genaccounts.AppModuleBasic{}))
```

Comment:
```suggestion
	rootCmd.AddCommand(testnet.TestnetCmd(ctx, cdc, app.ModuleBasics, genaccounts.AppModuleBasic{}))
```

Reply:
used to be what you're suggesting but causes linting error due to redundant naming aka the linter hates `youpackage.Yourpackage`

Reply:
See my comment below on this testnet logic living in it's own module. ie. I don't believe it warrants existing in its own module and hence can live somewhere else (maybe under `x/genutils`) and thus it would not have the `testnet` package name, but would have a `*/client/cli` package name 👍